run-drats.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
This is the script we've been using to run the DRATS on our local workstation. We just included it in the PR in case you want to include it or something similar in your `scripts` dir for other teams to use.

Reply:
We have a very similar script but for using in the ci. We will pull this into our scripts dir. Thanks!